packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -479,8 +664,35 @@ export const outOfOfficeEntriesList = async ({ ctx }: { ctx: { user: NonNullable
         },
       },
       notes: true,
+      user: fetchTeamMembersEntries
+        ? {
+            select: {
+              id: true,
+              username: true,
+              email: true,
+              avatarUrl: true,
+            },
+          }
+        : false,
+    },
+    orderBy: {
+      start: "desc",
```

Comment:
```suggestion
      start: "asc",
```

I think this should be ascending, closest ooo should be shown first

Reply:
ok